src/SprykerSdk/Spryk/Console/SprykDumpConsole.php

```diff
@@ -7,19 +7,27 @@
 
 namespace SprykerSdk\Spryk\Console;
 
+use Symfony\Component\Console\Helper\FormatterHelper;
 use Symfony\Component\Console\Helper\Table;
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
 class SprykDumpConsole extends AbstractSprykConsole
 {
+    protected const COMMAND_NAME = 'spryk:dump';
+    protected const COMMAND_DESCRIPTION = 'Dumps a list of all Spryk definitions.';
+
+    public const ARGUMENT_SPRYK = 'spryk';
+
     /**
      * @return void
      */
     protected function configure()
     {
-        $this->setName('spryk:dump')
-            ->setDescription('Dumps a list of all Spryk definitions.');
+        $this->setName(static::COMMAND_NAME)
+            ->setDescription(static::COMMAND_DESCRIPTION)
+            ->addArgument(static::ARGUMENT_SPRYK, InputOption::VALUE_OPTIONAL, 'Name of a specfic Spryk which its options should be dumped.');
```

Comment:
Please, change the text here to `Name of a specific Spryk for which the options should be dumped.`